### PR TITLE
Modified Last Years Highlights

### PR DESCRIPTION
**Pull request by sarvaSanjay**
> Ticket: https://app.clickup.com/t/86b1wk6hn

Updated with latest events and their links.
Also could you specify a date for HTSL? Not mentioned in ticket.

### CODE REVIEW

File: src/Assets/Lists/allEvents.js [hunk 1]
```diff
@@ -58,6 +58,25 @@ export const allEvents = [
...
         highlightEvent: true,
     },
+    {
+        EventName: "Technical Interview Night",
+        displayDate: "October 3, 2024",
+        date: "2024/10/03",
+        highlightEvent: false,
+    },
+    {
+        EventName: "NewHacks 2024",
```
Comment (li-ruihao) [inline src/Assets/Lists/allEvents.js:68]: For this, delete the _NewHacks 2024.jpg_ under /src/Assets/Images/events, and use the one from 2023, we don't need a duplicate one.

File: src/Assets/Images/events/NewHacks 2024.jpg
Comment (li-ruihao) [inline src/Assets/Images/events/NewHacks 2024.jpg:1]: please delete this file.

File: package-lock.json
Comment (li-ruihao) [inline package-lock.json:1]: please do not commit this file, uncommit this file please, thanks

**Review by li-ruihao — CHANGES_REQUESTED** [review]

File: src/Assets/Images/events/NewHacks 2024.jpg
Comment (sarvaSanjay) [reply src/Assets/Images/events/NewHacks 2024.jpg:1]: I don't quite think that's feasible. Every event's image at least for now is mapped by event name. I would rather suggest deleting the older images.

Comment (li-ruihao) [reply src/Assets/Images/events/NewHacks 2024.jpg:1]: I see, yea then leaving it there is fine, however let's not delete previous year images

**Review by li-ruihao — APPROVED** [review]

**Pull request merged**